### ISSUE TITLE
Menu goes outside when closer to edges.

### ISSUE THREAD

**Issue by vkbansal** [vkbansal/react-contextmenu#1]
![image](https://cloud.githubusercontent.com/assets/5930351/8508951/0ccf587c-22a5-11e5-8ea8-7c9ddfce2802.png)

Need to fix the display point calculation.

**Comment by vkbansal** [comment]
http://stackoverflow.com/questions/1841124/find-the-potential-width-of-a-hidden-element
> The width of an element that has CSS visibility: hidden is measurable

Possible solution
```css
.context-menu > .dropdown-menu {
    display: block;
    visibility: hidden;
}

.context-menu.open > .dropdown-menu {
    visibility: visible;
}
```

**Comment by vkbansal** [comment]
fixed in #6
